src/packagedcode/rubygems.py

```diff
@@ -647,93 +626,66 @@ def normalize(gem_data, known_fields=known_fields):
     )
 
 
-def parse_spec(location):
-    pass
-
-
-class GemSpec(object):
+def build_packages_from_gemspec(location):
     """
-    Represent a Gem specification.
+    Return RubyGem Package from gemspec file.
     """
+    gemspec_obj = ParseSpec()
+    gemspec_data = gemspec_obj.parse_spec(location)
+    
+    name = gemspec_data.get('name')
+    version = gemspec_data.get('version')
+    homepage_url = gemspec_data.get('homepage_url')
+    summary = gemspec_data.get('summary')
+    description = gemspec_data.get('description')
+    if len(summary)>len(description):
+        description = summary
+
+    declared_license = gemspec_data.get('license')
+    if declared_license:
+        declared_license = declared_license.split(',')
+
+    author = gemspec_data.get('author') or []
+    email = gemspec_data.get('email') or []
+    parties = []
+    if author:
+        parties.append(
+            models.Party(
+                type=models.party_person,
+                name=','.join(author),
```

Comment:
If author is a list, create as many Party as we have author.
Same for email: and since we cannot be sure that an author matches an email based on aligned lists, you should create one different Party for each author and each email.

Reply:
I will change this :)